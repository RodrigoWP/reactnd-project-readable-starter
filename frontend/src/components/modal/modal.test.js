'use strict'

import React from 'react'
import { shallow } from 'enzyme'
import { Modal } from '.'

it('Should render correctly', () => {
  const component = (
    <Modal />
  )

  expect(shallow(component)).toMatchSnapshot()
})
