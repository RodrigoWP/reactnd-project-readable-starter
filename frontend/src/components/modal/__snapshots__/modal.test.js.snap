// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<div
  className=""
>
  <div />
  <div>
    <div>
      <div>
        <img
          src=""
        />
      </div>
      <div>
        <span />
      </div>
      <div />
    </div>
  </div>
</div>
`;
